### PR TITLE
Fix: disable cmd+L keybinding in smartSearchField

### PR DESCRIPTION
**Pull request by attfarhan**
Fixes https://github.com/sourcegraph/sourcegraph/issues/9125.

### CODE REVIEW

File: web/src/search/input/MonacoQueryInput.tsx [hunk 1]
```diff
@@ -126,7 +126,11 @@ function addSouregraphSearchCodeIntelligence(
     return subscriptions
 }
 
-const NOOP_KEYBINDINGS = [Monaco.KeyMod.CtrlCmd | Monaco.KeyCode.KEY_F, Monaco.KeyMod.CtrlCmd | Monaco.KeyCode.Enter]
+const NOOP_KEYBINDINGS = [
+    Monaco.KeyMod.CtrlCmd | Monaco.KeyCode.KEY_F,
+    Monaco.KeyMod.CtrlCmd | Monaco.KeyCode.Enter | Monaco.KeyCode.KEY_L,
+    Monaco.KeyMod.CtrlCmd,
```
Comment (mrnugget) [inline web/src/search/input/MonacoQueryInput.tsx]:
```suggestion
    Monaco.KeyMod.CtrlCmd | Monaco.KeyCode.Enter,
    Monaco.KeyMod.CtrlCmd | Monaco.KeyCode.KEY_L,
```

Comment (mrnugget) [reply web/src/search/input/MonacoQueryInput.tsx]: Looking at the "before" code I think it should be this, but I don't know how Monaco works :)

**Comment by attfarhan** [comment]
Ugh I just realized I had interactive mode on when testing. You're comment is correct, but this doesn't actually fix it, it just disables cmd+L completely... let me see if there's a better fix

**Comment by attfarhan** [comment]
Ok it appears there's no easy solution for this... Let me close this PR and then discuss with Loic on the issue.

**Pull request closed without merging**